lib/spack/docs/environments.rst

```diff
@@ -6,7 +6,7 @@
 .. _environments:
 
 =========================
-Environments (spack.yaml)
+Environments (``spack.yaml``)
```

Comment:
I'd remove this for consistency with other sections. We can do this minor tweak on all the sections reporting the yaml file in a separate PR.
```suggestion
Environments (spack.yaml)
```